addon/index.js

```diff
@@ -10,7 +9,7 @@ export function initialize(container, config) {
 
   if (get(config, 'sentry.development') === true) {
     if (get(config, 'sentry.debug') === true) {
-      Ember.Logger.info('`sentry` is configured for development mode.');
+      console.info('`sentry` is configured for development mode.'); // eslint-disable-line no-console
```

Comment:
I guess we should probably disable the `no-console` rule globally instead of overriding it for every line where we use it 🤔